### ISSUE TITLE
Update to 114 needs some changes, and possibly needs to be skipped

### ISSUE THREAD

**Issue by stephanema1** [jcefmaven/jcefbuild#10]
Just downloaded: https://github.com/jcefmaven/jcefbuild/releases/download/1.0.47/windows-amd64.tar.gz

unpacked it on my windows 10 PC started the run.bat and got:

    C:\tooling\programs\jcef-java\114>run.bat

    C:\tooling\programs\jcef-java\114>java -cp "./bin;./bin/*" -Djava.library.path=./bin/lib/win64 tests.detailed.MainFrame
    Offscreen rendering disabled
    Exception in thread "main" java.lang.UnsatisfiedLinkError: chrome_elf (Not found in java.library.path)
            at java.base/java.lang.ClassLoader.loadLibraryWithPath(ClassLoader.java:1742)
            at java.base/java.lang.ClassLoader.loadLibraryWithClassLoader(ClassLoader.java:1694)
            at java.base/java.lang.System.loadLibrary(System.java:615)
            at org.cef.SystemBootstrap$1.loadLibrary(SystemBootstrap.java:24)
            at org.cef.SystemBootstrap.loadLibrary(SystemBootstrap.java:36)
            at org.cef.CefApp.<init>(CefApp.java:145)
            at org.cef.CefApp.getInstance(CefApp.java:221)
            at tests.detailed.MainFrame.<init>(MainFrame.java:105)
            at tests.detailed.MainFrame.main(MainFrame.java:76)`

**Comment by FriwiDev** [comment]
Yeah, the newest release seems to be broken. I started a build yesterday, but sadly the build for linux-arm(32) is not even available by spotify, the macos builds fail and the windows builds could be broken - I have not tested them. I deleted the release for now and hope for the next jcef version to fix some things. I might need to perform some changes to the macos signing to get those builds to work, but the rest is basically a jcef issue.
